design/ceph/cephx-keyring-rotation.md

```diff
@@ -307,6 +309,14 @@ Ceph daemon to ensure key rotation is applied:
 With separated daemon key info tracking, the status will look like so:
 
 ```yaml
+spec:
+  security:
+    cephx:
+      daemon: {}
+      csi: {}
+      # I don't really like having this on the CephCluster config. I wonder if we could move this
+      # to the CephRBDMirror CR without messing up users
```

Comment:
`client.rbd-mirror-peer` user and key are created even before the `cephRBDMirror` peer or any of the `cephBlockPools` is created.  So I think its better if the rotation of this key (client.rbd-mirror-peer) is handled a `cephcluster` spec setting. 

Don't think we have a better alternative here.

Reply:
Overall, I think I agree.

We **could** have the rotation setting be on the CephRBDMirror resource, although that would get complicated to understand as a dev or user, which I don't like.

Based on Madhu's suggestion, we may move management of this key to CephRBDMirror in the future anyway, and then have to deprecate this section, which I wouldn't like.

At the end of the day, I don't prefer either option, but probably it's best to go with the simplest solution now and then change in the future if we decide we want to.